openhands/server/routes/public.py

```diff
@@ -65,4 +65,4 @@ async def get_config() -> dict[str, Any]:
     Returns:
         dict[str, Any]: The current server configuration.
     """
-    return server_config.get_config()
+    return server_config.get_config().model_dump()
```

Comment:
```suggestion
    return server_config.get_config()
```

I think FastAPI does this conversation automatically